update_jets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+
+if [ -z "$1" ]; then
+  echo "\$1 parameter must be the simplicity directory"
+  echo "\$2 parameter (optional) can be the revision to check out"
```

Comment:
In 8a8be85d857af7a55d2c8076f96eaee5957c524b:

IMO we shouldn't support specifying a revision. It's surprising that a script called `update-jets.sh` would cd into the source directory and then change files there.

Also, we should have the jet script default to `$(git rev-parse --show-toplevel)/simplicity-sys/depend/simplicity/` if it is not supplied, since going forward that's likely to be the common case.

Reply:
Ah, I see, the bundled version of simplicity does not include any Haskell code. That's a little unfortunate because it makes it hard to ensure you're running `update-jets.sh` with the same version of the Simplicity code as is bundled. I guess, it's fine for now.

Reply:
Alternate strategy: if $2 is not provided, read the commit ID from `"$(git rev-parse --show-toplevel)"/simplicity-sys/depend/simplicity-HEAD-revision.txt`. In fact, even if $2 **is** provided, warn if the provided value differs from this value.

Then in the source directory, rather than doing `git checkout $REV`, we should check that the output of `git rev-parse HEAD` matches `$REV`, and output an error if they don't match.

Reply:
Agreed